### DISCUSSION TITLE
Folder location for "Export Server Timings Details"

### DISCUSSION THREAD

**Discussion by SharmaAntriksh** [DaxStudio/DaxStudio#1238] [General]
Where is this folder located?

![image](https://github.com/DaxStudio/DaxStudio/assets/73054674/40a13eee-a0e2-4818-8e41-eae43fd9cea9)

**Comment by dgosbell** [comment]
When you click export button after enabling this option it displays a folder picker where you can choose where put the exported data.

**Reply by SharmaAntriksh** [reply]
Thanks!! I thought it does that automatically for each run like log files 🤣
